laborator/content/interactiune-c-assembly/09-export-fix/a-func/ops.h

```diff
@@ -0,0 +1,11 @@
+#ifndef OPS_H_
```

Comment:
```suggestion
/* SPDX-License-Identifier: BSD-3-Clause */

#ifndef OPS_H_
```
Same for all `.h` files.